runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -668,7 +672,7 @@ static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode, TR::SymbolRefer
             }
          }
    else if (storeValueNode->getOpCode().isCall()
-            && storeValueNode->getSymbolReference() == loadFlattenableElementSymRef)
+            && (symRefTab->isNonHelper(storeValueNode->getSymbolReference(), loadFlattenableArrayElementNonHelper) || (storeValueNode->getSymbolReference() == loadFlattenableElementHelperSymRef)))
```

Comment:
This line is very long.  I would suggest starting a new line at the `||` operator

Reply:
Addressed in 446e692